tests/e2e/specs/confluent.spec.ts

```diff
@@ -1,12 +1,18 @@
+import { stubAllDialogs } from "electron-playwright-helpers";
 import { test } from "../baseTest";
-
+import { Tag } from "../tags";
 import { openConfluentExtension } from "./utils/confluent";
 import { login } from "./utils/confluentCloud";
 
 test.describe(() => {
-  test("should load the extension properly", async ({ page }) => {
-    await openConfluentExtension(page);
-  });
+  test(
+    "should load the extension properly",
+    { tag: [Tag.Smoke] },
+    async ({ page, electronApp }) => {
+      await stubAllDialogs(electronApp);
```

Comment:
We may need to handle this at the global level depending on how the next branch(es) behave with more tests running in CI, but for now this is all we need.